src/ast/stmt.rs

```diff
@@ -1,8 +1,33 @@
-use super::expr::Expr;
+/*
+ * BNF which statement parser can accept for now
+ *
+ * <compound> ::=  { {<decl>}* {<stmt>}+ }
+ *
+ * <stmt> ::= <compound>
+ *          | <jump>
+ *
+ * <decl> ::= {<decl-specifier>}+ {<init-declarator>}*};
```

Comment:
今はこれだけで十分でしょう。
```suggestion
 * <decl> ::= <type-specifier> <identifier>;
```
この変更に伴って、いくつかの型や実装が必要なくなると思います。